schemas/mod_links.graphql

```diff
@@ -0,0 +1,56 @@
+### Types
+
+scalar ModLinkID
+
+type ModLink {
+    id: ModLinkID!
+    ModVersionLinkID: String!
+    platform: String!
+    asset: String!
+    size: Int
+    hash: String
+}
+
+enum ModLinkFields {
+    platform
+}
+
+type GetModLinks {
+    arch: [ModLink!]!
+}
+
+### Inputs
+
+input ModLinkFilter {
+    limit: Int
+    offset: Int
+    order_by: ModLinkFields
+    order: Order
+    search: String
+    ids: [String!]
+}
+
+input NewModLink {
+    id: ModLinkID
+    ModVersionLinkID: String!
+    platform: String!
+    key: String!
+    size: Int
+    hash: String
+}
+
+### Queries
+
+extend type Query {
+    getModLink(id: ModLinkID!): ModLink
+    getModLinks(filter: ModLinkFilter): GetModLinks!
+
+    getModDownload(ModVersionLinkID: String!, platform: String!): String!
+}
+
+### Mutations
+
+extend type Mutation {
+    createModLink(mod_link: NewModLink!): ModLink @isLoggedIn
+    deleteModLink(ModLinksId: ModLinkID!): Boolean! @canEditMod(field: "versionId") @isLoggedIn
+}
```

Comment:
It should not be possible to modify version files after initial upload.

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/8ea4f2b30882e41af00e95ce37ab41b3a4ebd4e3